cppcache/test/CMakeLists.txt

```diff
@@ -55,7 +56,7 @@ add_executable(apache-geode_unittests
   util/synchronized_setTest.cpp
   util/TestableRecursiveMutex.hpp
   util/chrono/durationTest.cpp
-)
+        PdxTypeTest.cpp)
```

Comment:
Looks like we are adding this file twice.

Reply:
Right, CLion added it for me and it added it manually

Reply:
We should check and see if `tests/cpp/testobject/PdxType.*` are in use anywhere.  We may get lucky and find that they can be deleted.

Reply:
> We should check and see if `tests/cpp/testobject/PdxType.*` are in use anywhere. We may get lucky and find that they can be deleted.

It's used in several old and some new integration tests.